modin/experimental/core/io/sql/sql_dispatcher.py

```diff
@@ -72,7 +72,7 @@ def _read(
             message = "Defaulting to Modin core implementation; \
                 'partition_column', 'lower_bound', 'upper_bound' must be different from None"
             warnings.warn(message)
-            return cls.base_io.read_sql(
```

Comment:
In the current implementation, `base_io` is `BaseIO` instead of `PandasOnRayIO`.

Reply:
TODO: Check for other experimental features to see if this change is necessary for them.

Reply:
Not necessary